components/forms/Button/DeleteButton.tsx

```diff
@@ -33,7 +34,7 @@ export const DeleteButton = (props: DeleteButtonProps): React.ReactElement => {
               return;
             }
           } catch (e) {
-            console.error(e);
+            logMessage.error(e as string);
```

Comment:
Negligible `logMessage.error(e as string);` should be replaced by  `logMessage.error(e as Error);` so it removes the side effect of logging a `string` instead of an `Error object`.